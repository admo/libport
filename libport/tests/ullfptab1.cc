#define URBI_UFLOAT_LONG_LONG
#define URBI_UFLOAT_TABULATED
#include "common_ufloat1.cc"
