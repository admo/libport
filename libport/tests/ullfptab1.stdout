f < g: true
g > f: true
f == f: true
f != g: true
f == g: true
f <= g: true
f >= g: true
